Lib/Common/Shared/Protocol/Constants.cs

```diff
@@ -1495,7 +1521,7 @@ static HeaderConstants()
             /// Current storage version header value.
             /// Every time this version changes, assembly version needs to be updated as well.
             /// </summary>
-            public const string TargetStorageVersion = "2019-02-02";
+            public const string TargetStorageVersion = "2019-07-07";
```

Comment:
FYI: This change should be called out in changelog when we get to release.

Reply:
Definitely.. Only I didn't update change log in this PR.